public/css/common.less

```diff
@@ -403,3 +403,11 @@ div.show-more {
 form[name="form_confirm_removal"] {
   text-align: center;
 }
+
+.affected-objects {
+  display: inline-flex;
+  align-items: baseline;
+  background-color: @state-critical;
+  color: @default-text-color-inverted;
```

Comment:
Please don't use default less variables if there is an alternative non-default: `@text-color-inverted`

Reply:
this is a candidate for a draft issue